ballerina/private_public_key.bal

```diff
@@ -170,6 +170,19 @@ public isolated function decodeRsaPrivateKeyFromKeyFile(string keyFile, string?
     'class: "io.ballerina.stdlib.crypto.nativeimpl.Decode"
 } external;
 
+# Decodes the RSA private key from the given private key content as a byte array.
```

Comment:
I guess we can add support for key types other than rsa in the future right?

Reply:
Yes, due to time limitations, it was implemented only for the RSA type. We will be introducing this to EC as well. Created issue :- https://github.com/ballerina-platform/ballerina-library/issues/6581